ecm-wcm-extension/src/main/webapp/attachments-selector/css/attachments.less

```diff
@@ -1,1183 +1,1230 @@
 @import "variables.less";
 @import "mixins.less";
 
-.attachment {
```

Comment:
the modification of existing less files can impact old attachment components, thus I advice to define new CSS file for new component and avoir changing the ones used for old one, WDYT ?

Reply:
I just added the old app parent class name to not affect our new app.